helper/certutil/types.go

```diff
@@ -45,6 +45,16 @@ const (
 	TLSClient
 )
 
+//BlockType indicates the serialization format of the key
+type BlockType string
+
+//Well-known formats
+const (
+	PKCS1Block BlockType = "RSA PRIVATE KEY"
+	PKCS8Block BlockType = "PRIVATE KEY"
+	ECBlock    BlockType = "EC PRIVATE KEY"
+)
```

Comment:
A high level comment here: PKCS1 isn't always RSA (that's kind of the point of PKCS8 trying to carry type info with it, right?). So I think it's more accurate to use `PKCS1RSABlock` and `PKCS1ECBlock`, even though it's uglier.